docs/man1/compareECL.1

```diff
@@ -1,17 +1,19 @@
-.TH COMPAREECL "1" "October 2023" "compareECL 2023.10" "User Commands"
+.TH COMPAREECL "1" "October 2024" "compareECL" "User Commands"
 .SH NAME
 compareECL \- Comparator for Eclipse files
 .SH SYNOPSIS
 .B convertECL
-[\fI\,OPTIONS\/\fR] \fI\,ECL_DECK_FILENAME_1\/\fR \fI\,ECL_DECK_FILENAME_2\/\fR
+[\fI\,OPTIONS\/\fR] \fI\,ECL_DECK_FILENAME_1\/\fR
+\fI\,ECL_DECK_FILENAME_2\/\fR \fI\,ABSOLUTE_TOLERANCE\/\fR  \fI\,RELATIVE_TOLERANCE\/\fR
 .SH DESCRIPTION
 compareECL compares ECLIPSE files (restart (.RST), unified restart (.UNRST), initial (.INIT), summary (.SMRY), unified summary (.UNSMRY) or .RFT) and gridsizes (from .EGRID or .GRID file) from two simulations.
 The program takes four arguments:
 .PP
-1. Case number 1, reference case (full path without extension)
-2. Case number 2, test case (full path without extension)
-3. Absolute tolerance
-4. Relative tolerance (between 0 and 1)
+.nf
+- ECL_DECK_FILENAME_1, first reference case as full path, but without extension.
+- ECL_DECK_FILENAME_2, second reference case eith full path, but without extension.
```

Comment:
These are arguably not full paths since users may reasonably write something like
```sh
compareECL ../ref/CASE CANDIDATE 1.0e-3 5.0e-6
```
Maybe we should just say "path to case" or something along those lines?